manim/mobject/opengl/opengl_image_mobject.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+__all__ = [
+    "OpenGLImageMobject",
+]
+
+import numpy as np
+from PIL import Image
+
+from manim.mobject.opengl.opengl_surface import OpenGLSurface, OpenGLTexturedSurface
+from manim.utils.images import get_full_raster_image_path
+
+
+class OpenGLImageMobject(OpenGLTexturedSurface):
+    def __init__(
+        self,
+        filename_or_array: str | np.ndarray,
+        width: float = None,
+        height: float = None,
+        opacity: float = 1,
+        gloss: float = 0,
+        shadow: float = 0,
+        **kwargs,
```

Comment:
I've managed to implement this; though, there are a few minor problems due to extra requirements for moderngl to process the texture, i.e., the `image_mode="L"` doesn't return a greyscale picture because moderngl needs the `swizzle` parameter specifically passed. The rest ("HSV" or "YCbCr") works as intented. 

Reference: 
[from the moderngl repo](https://github.com/moderngl/moderngl/issues/112#issuecomment-310133422)